README.rst

```diff
@@ -24,10 +24,10 @@ In your Scrapy ``settings.py``:
 1. Enable HTTP cache with ``HTTPCACHE_ENABLED = True``
 2. Set the cache expiration (in seconds), for example, with ``HTTPCACHE_EXPIRATION_SECS = 600``
 3. Set ``MemcachedCacheStorage`` as your cache storage with ``HTTPCACHE_STORAGE = 'scrapy_memcached_cache.MemcachedCacheStorage'``
-4. Set the location where your Memcached is running, for example, with ``MEMCACHED_LOCATION=localhost:11211``
+4. Set the location where your Memcached is running, for example, with ``MEMCACHED_LOCATION='localhost:11211'``
```

Comment:
Following [Black](https://github.com/ambv/black) style (mentioned a few lines above), the proper way would be `MEMCACHED_LOCATION = "localhost:11211"` (instead of `MEMCACHED_LOCATION='localhost:11211'`). Would you mind editing it? Many thanks ; )